test_requirements/requirements_base.txt

```diff
@@ -1,19 +1,19 @@
-coverage<5
-python-coveralls==2.5.0
+coverage>=4
+python-coveralls>2.5.0
 unittest-xml-reporting==1.11.0
 Pillow==5.2.0
-django-treebeard>=4.3
+django-treebeard>=4.3,<4.5
 argparse
 dj-database-url
 djangocms-admin-style>=1.5
-django-sekizai>=0.7
-django-classy-tags>=0.7.2
-https://github.com/divio/djangocms-text-ckeditor/archive/master.zip
+django-sekizai>=0.7,<=1.10
```

Comment:
these should be pinned in ``django-2.2.txt`` instead as 3.0 and 3.1 do not support Python 2.7

(apples for classy-tags and ckeditor as well

Reply:
@Bernardvdv As Angelo has requested we should add this dependancy in django-2.2.txt, I think we then need to add this: `django-sekizai>=0.7` to the other django_3.x files as a duplication will cause issues.